ios/HrRecruit/AppDelegate.m

```diff
@@ -11,11 +11,14 @@
 
 #import <React/RCTBundleURLProvider.h>
 #import <React/RCTRootView.h>
+#import "RNFIRMessaging.h"
```

Comment:
why are all such files part of git. we should not any of such files . ios or android specific